### DISCUSSION TITLE
Download DUALSPHYSics_V5.4

### DISCUSSION THREAD

**Discussion by hongshanma** [DualSPHysics/DualSPHysics#278] [DUALSPHYSICS_V5.4_BETA]
Dear Developers，
I would like to download DUALSPHYSics_V5.4，but this link is failed.
![image](https://github.com/user-attachments/assets/a3daf33a-96cf-426f-95fd-3ea1535bfce6)

Looking forward to your reply, thank you very much!

**Comment by alexbexe** [comment]
DualSPHysics_v5.4_BETA was only provided to the attendees of last DualSPHysics Workshop in Bari.
However, the official release of v5.4 is expected for next March 2025

Regards
Alex

**Comment by MaryamBC** [comment]
Hello Alex,

 Could you please provide an update on the expected date for the public release of v5.4?
Thanks
